hw/ip/dcd/dcd.core

```diff
@@ -0,0 +1,62 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:dcd:1.0"
+description: "dcd"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:constants:top_pkg
+      - lowrisc:prim:all
+      - lowrisc:ip:tlul
+      - lowrisc:systems:ast_wrapper_pkg
+    files:
+      - rtl/dcd_reg_pkg.sv
+      - rtl/dcd_reg_top.sv
+      - rtl/dcd_pkg.sv
+      - rtl/dcd_fsm.sv
+      - rtl/dcd_intr.sv
+      - rtl/dcd_core.sv
+      - rtl/dcd.sv
+    file_type: systemVerilogSource
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+    file_type: waiver
+
+  files_veriblelint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+
+parameters:
+  SYNTHESIS:
+    datatype: bool
+    paramtype: vlogdefine
+
+targets:
+  default: &default_target
+    filesets:
+      - tool_ascentlint  ? (files_ascentlint_waiver)
+      - files_rtl
+    toplevel: dcd
+
+
+  lint:
```

Comment:
similar comment as on https://github.com/lowRISC/opentitan/pull/4900, please replace by the snippet I posted there